superset/views/annotations.py

```diff
@@ -103,7 +103,7 @@ class AnnotationLayerModelView(SupersetModelView):  # pylint: disable=too-many-a
     add_title = _("Add Annotation Layer")
     edit_title = _("Edit Annotation Layer")
 
-    list_columns = ["name", "descr"]
+    list_columns = ["id", "name", "descr"]
```

Comment:
This will make the CRUD view slightly uglier with the `id`, but is necessary based on discussions with @dpgaspar